src/wallet/wallet.cpp

```diff
@@ -2120,11 +2137,12 @@ void CWallet::ResendWalletTransactions(interfaces::Chain::Lock& locked_chain, in
     if (fFirst) return;
 
     // Only do it if there's been a new block since last time
-    if (nBestBlockTime < nLastResend) return;
+    if (m_best_block_time < nLastResend) return;
     nLastResend = GetTime();
 
     int relayed_tx_count = 0;
 
+    auto locked_chain = chain().lock();
     { // cs_wallet scope
```

Comment:
style-nit in f7790b3ad4c0cd5318841e30224c0d3dea0a3dca:

any reason to scope cs_wallet, but not cs_main?

Reply:
fixed